ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ipv6-neighbors.j2

```diff
@@ -0,0 +1,27 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - ipv6 neighbors #}
+{% if ipv6_neighbor.cache is arista.avd.defined or ipv6_neighbor.persistent is arista.avd.defined or ipv6_neighbor.static_entries is arista.avd.defined %}
+!
+{%     if ipv6_neighbor.cache is arista.avd.defined(true) %}
+ipv6 neighbor cache persistent
+{%     endif %}
+{%     if ipv6_neighbor.persistent is arista.avd.defined(true) %}
+ipv6 neighbor persistent refresh-delay {{ ipv6_neighbor.refresh_delay | arista.avd.default('600') }}
+{%     endif %}
+{%     if ipv6_neighbor.static_entries is arista.avd.defined %}
+{%         for neighbor in ipv6_neighbor.static_entries %}
```

Comment:
We can use natural_sort('ipv6_address')

Reply:
EOS does not sort IPv6 addresses like natural sort. 
Output from EOS
```
ipv6 neighbor vrf MGMT 11:22:33:44:55:66:77:88 Ethernet1 01:01:02:02:03:03
ipv6 neighbor ::ffff:191.1.56.10 Loopback99 aa:af:12:34:bc:bf
ipv6 neighbor ::ffff:192.1.56.10 Vlan11 12:23:34:56:67:47
ipv6 neighbor ::ffff:192.1.56.10 Loopback9 aa:af:12:34:01:23
ipv6 neighbor ::ffff:192.1.56.101 Vlan111 12:23:34:56:67:47
ipv6 neighbor 11:11:33:44:55:66:77:88 Ethernet2 01:11:02:23:03:33
```
cc: @ClausHolbechArista

Reply:
I don't have a solution for this. Let's skip the sorting for now.